discovery-provider/src/utils/config.py

```diff
@@ -97,3 +97,7 @@ def _load_item(self, section_name, key):
     for static_item in shared_config.items(section):
         static_key = static_item[0]
         env_config_update(shared_config, section, static_key)
+
+# Set db_read_replica url to same as db url if none provided
+if 'url_read_replica' not in shared_config['db'] or not shared_config['db']['url_read_replica']:
```

Comment:
not sure this works for current_app.config access...best to move into config parser maybe

Reply:
good looks